cpp/models/ide_secir/model.h

```diff
@@ -212,10 +205,11 @@ class Model
      *
      * @returns 0 if the initialization method has not yet been selected,
```

Comment:
```suggestion
     * @brief Returns the number associated with the method selected automatically for initialization.
     
     * @returns 0 if the model has not yet been initialized and the method has not yet been selected,
```

Ich kann Zeile 211 hier nicht suggesten weil dort kein Change ist, aber bitte auch ändern.